packages/shared/src/utils/cfd/cfd.ts

```diff
@@ -274,20 +274,34 @@ export const getAuthenticationStatusInfo = (account_status: GetAccountStatus) =>
 
     //vanuatu-maltainvest
 
-    const poi_verified_for_vanuatu_maltainvest = [onfido_status, manual_status].includes('verified');
+    const poi_verified_for_vanuatu_maltainvest = [
+        onfido_status,
+        manual_status,
+        ...(is_idv_supported ? [idv_status] : []),
+    ].includes('verified');
     const poi_acknowledged_for_vanuatu_maltainvest =
         (onfido_status && acknowledged_status.includes(onfido_status)) ||
-        (manual_status && acknowledged_status.includes(manual_status));
-
-    const poi_pending_for_vanuatu_maltainvest =
-        onfido_status &&
-        manual_status &&
-        [onfido_status, manual_status].includes('pending') &&
-        !poi_verified_for_vanuatu_maltainvest;
+        (manual_status && acknowledged_status.includes(manual_status)) ||
+        (is_idv_supported && idv_status && acknowledged_status.includes(idv_status));
+
+    const poi_pending_for_vanuatu_maltainvest = !is_idv_supported
+        ? onfido_status &&
+          manual_status &&
+          [onfido_status, manual_status].includes('pending') &&
+          !poi_verified_for_vanuatu_maltainvest
+        : onfido_status &&
+          manual_status &&
+          idv_status &&
+          [onfido_status, manual_status, idv_status].includes('pending') &&
+          !poi_verified_for_vanuatu_maltainvest;
```

Comment:
These changes will effect maltainvest. could you please double check?

Reply:
there's a check to include idv_status if idv is supported otherwise the check the old implementation will be used and it's working as expected. Malta invest isn't supported by idv, so the implementation will fallback to previous implementation without idv_status

Reply:
For DIEL south africa clients, idv is supproted. But we need only onfido to verify thier account. This will break the flow.

Reply:
For DIEL south africa clients, idv is supproted. But for eu- maltainvest section, we need only onfido to verify thier account. This will break the flow.